config/ampache.cfg.php.dist

```diff
@@ -756,12 +756,18 @@ site_charset = UTF-8
 ;#########################################################
 
 ; This setting will silently create an ampache account
-; for anyone who can login using ldap (or any other login
+; for anyone who can login using LDAP (or any other login
 ; extension). The default is to create new users as guests
 ; see auto_user config option if you would like to change this
 ; DEFAULT: false
 ;auto_create = "false"
 
+; This setting will silently update an ampache account's
+; informations for anyone who can login using LDAP
+; (or any other login extension).
+; DEFAULT: false
+;auto_update = "false"
```

Comment:
Good to have an option for this but it can mislead administrators because of the other `autoupdate` setting. Would be better to rename it or prefix it with `ldap_`